validate.js

```diff
@@ -3,7 +3,7 @@ var ConfigError = require('./error')
 var regex = new RegExp('^[a-zA-Z0-9_.-]{3,255}$');
 
 var validate = function (options, field) {
-  if (!regex.test(options['field'] || '')) {
+  if (!regex.test(options[field] || '')) {
```

Comment:
not enough need to replace below by:
module.exports.config = function (options) {
    return validate(options.source, 'tableName') && validate(options.destination, 'tableName') // check both source and destination
}

Reply:
@najimou thanks, check `.tableName` now.